src/components/Navigation.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useContext } from 'react';
+import Accordion from 'react-bootstrap/esm/Accordion';
+import Nav from 'react-bootstrap/esm/Nav';
+import { appContext } from '../context/AppProvider';
+
+export const Navigation: FC = () => {
+  const context = useContext(appContext);
+
+  return (
+    <Nav className="flex-column">
+      <Nav.Link href="/">{context.ecoverse.name}</Nav.Link>
+      <Accordion defaultActiveKey="0">
+        <Accordion.Toggle as={Nav.Link} variant="light" eventKey="0">
+          Challange
+        </Accordion.Toggle>
+        <Accordion.Collapse eventKey="0">
+          <div style={{ marginLeft: '10px' }}>
+            <Nav.Link href="/governance">Governance</Nav.Link>
```

Comment:
Spelling should be with an e i.. governance.

That said, for now I would like to have a different set of topics under this heading: Context, Community, Opportunities, Projects - will refine more later